src/interface/raftex.thrift

```diff
@@ -7,6 +7,21 @@ namespace cpp nebula.raftex
 
 include "common.thrift"
 
+enum Role {
+    LEADER      = 1, // the leader
+    FOLLOWER    = 2; // following a leader
+    CANDIDATE   = 3; // Has sent AskForVote request
+    LEARNER     = 4; // It is the same with FOLLOWER,
+                     // except it does not participate in leader election
+}
+
+enum Status {
+    STARTING            = 0; // The part is starting, not ready for service
+    RUNNING             = 1; // The part is running
+    STOPPED             = 2; // The part has been stopped
+    WAITING_SNAPSHOT    = 3; // Waiting for the snapshot.
+}
+
```

Comment:
These enum types have been defined in following source file, could you unify the usage of these enum types?

https://github.com/vesoft-inc/nebula/blob/b7c690114016ce855df000abcc376f8903b30b12/src/kvstore/raftex/RaftPart.h#L239-L252

Reply:
@yixinglu updated, PTAL